client/src/components/invitation/sendInvite.component.js

```diff
@@ -5,7 +5,7 @@ import BetterChips from "../addons/betterChips.componenet.js";
 import M from "materialize-css";
 import axios from "axios";
 
-const SendInvite = props => {
+const SendInvite = (props) => {
```

Comment:
Here is an axample where you have added those parentheses. (according to previous comment)

Reply:
Either are correct, because it's a single argument. 
Anyway this was changed automatically on save by my linter package, which I had updated before the commit. Somehow I missed that. 
But in the end it changes nothing AFAIK.